build.gradle

```diff
@@ -336,7 +336,7 @@ dependencyManagement {
 dependencies {
 
     implementation group: 'com.github.hmcts', name: 'et-common', version:'1.5.7'
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'3.6.6'
+    implementation group: 'com.github.hmcts', name: 'et-data-model', version:'3.6.8-beta'
```

Comment:
I think you're okay to use a non beta release

Reply:
I have created the 3.6.8 Release